notifications.json

```diff
@@ -0,0 +1,861 @@
+                                        "mobile": {
+                                            "type": "object",
+                                            "properties": {
+                                                "status": {
+                                                    "type": "string"
+                                                }
+                                            }
+                                        },
+                                        "web": {
+                                            "type": "object",
+                                            "properties": {
+                                                "status": {
+                                                    "type": "string"
+                                                }
+                                            }
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "",
+                        "schema": {
+                            "$ref": "#/definitions/message-output"
+                        }
+                    },
+                    "400": {
+                        "$ref": "#/responses/trait:apiErrors:400"
+                    },
+                    "401": {
+                        "$ref": "#/responses/trait:apiErrors:401"
+                    },
+                    "403": {
+                        "$ref": "#/responses/trait:apiErrors:403"
+                    },
+                    "404": {
+                        "$ref": "#/responses/trait:apiErrors:404"
+                    },
+                    "429": {
+                        "$ref": "#/responses/trait:apiErrors:429"
+                    }
+                }
+            },
+            "delete": {
+                "operationId": "deleteMessage",
+                "summary": "Delete Message",
+                "tags": [
+                    "Messages"
+                ],
+                "responses": {
+                    "204": {
+                        "description": ""
+                    },
+                    "400": {
+                        "$ref": "#/responses/trait:apiErrors:400"
+                    },
+                    "401": {
+                        "$ref": "#/responses/trait:apiErrors:401"
+                    },
+                    "403": {
+                        "$ref": "#/responses/trait:apiErrors:403"
+                    },
+                    "404": {
+                        "$ref": "#/responses/trait:apiErrors:404"
+                    },
+                    "429": {
+                        "$ref": "#/responses/trait:apiErrors:429"
+                    }
+                }
+            }
+        },
+        "/Messages": {
```

Comment:
the commentend line is the endpoint for the sending operation (it's a POST actually)